frontend/src/routes/Video.tsx

```diff
@@ -609,6 +637,7 @@ const MetadataTable: React.FC<MetadataTableProps> = ({ event }) => {
     return (
         <dl css={{
```

Comment:
Could you try (in a separate commit or whatever) to format this metadata thing like this?

```
Part of series:
    The best open cat videos
License:
    Pixabay License
Source:
    https://pixabay.com/videos/chicken-
    hen-full-hd-60-fps-animal-32471/
```

At least on small screens. I think this could look nice and saves us some horizontal space.

Reply:
Mh so I remember someone suggesting that this is not a great idea and label + data should be on one line. What was that about? I do see the reasoning as my suggested (and currently implemented) layout costs twice as much vertical space most of the time.

Reply:
I think you refer to #802. If we do put it back in one line, than we should of course also give the metadata box more horizontal space. Though maybe not quite 50%. Right now it is roughly 2:1, but maybe we can do like 3:2 or sth in that direction.

Reply:
Ah right yes. So let's maybe just keep it as is right now and then we can think about changing it later. The issue you linked will be kept open to remind us.